openhtf/util/units/__init__.py

```diff
@@ -0,0 +1,1170 @@
+# coding: iso-8859-1
+
+# Copyright 2015 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Units of measure for OpenHTF.
+
+Used to retrieve UNECE unit codes/suffixes as follows:
+
+from openhtf.utils.units import UNITS
```

Comment:
in my (admittedly limited) experience using this, it gets *real* annoying very quickly to have a long name for this, and if it's a sub-variable in a name, you can't even use import renaming to make it shorter, you have to use an assignment at the top of the file as an alias.

if you're all right with that, then that's fine, but expect a lot of `U = units.UNITS` at the top of files.